addons/website_sale/views/templates.xml

```diff
@@ -308,18 +308,18 @@
                             <t t-call="website_sale.products_breadcrumb">
                                 <t t-set="_classes" t-valuef="d-none d-lg-flex w-100 p-0 small"/>
                             </t>
-                            <div class="products_header btn-toolbar flex-nowrap align-items-center justify-content-between mb-3">
+                            <div class="products_header btn-toolbar flex-nowrap align-items-center justify-content-between gap-3 mb-3">
                                 <t t-if="is_view_active('website_sale.search')" t-call="website_sale.search">
                                     <t t-set="search" t-value="original_search or search"/>
                                     <t t-set="_form_classes" t-valuef="d-lg-inline {{'d-inline' if not category else 'd-none'}}"/>
                                 </t>
 
                                 <t t-call="website_sale.pricelist_list" t-cache="pricelist">
-                                    <t t-set="_classes" t-valuef="d-none d-lg-inline {{'ms-3' if is_view_active('website_sale.search') else ''}}"/>
+                                    <t t-set="_classes" t-valuef="d-none d-lg-inline"/>
```

Comment:
This is the expected result. I think `ms-auto` is missing on `o_wsale_apply_layout`.
![Capture d’écran 2023-10-05 à 16 31 00](https://github.com/odoo/odoo/assets/80679690/4c63af13-d42e-4001-a8c7-7467152cce4e)

The aim was also to clean up the margin calls on `o_wsale_apply_layout` button (on master 👇 ) :
![Capture d’écran 2023-10-05 à 16 23 35](https://github.com/odoo/odoo/assets/80679690/d442fa57-3484-4f68-889e-9f1e0f0c0d12)

Now I realize that the filter button shouldn't have `ms-3` since `gap-3` is already set on the parent. 
| Expectation | Reality |
|--------|--------|
| ![Capture d’écran 2023-10-05 à 16 25 36](https://github.com/odoo/odoo/assets/80679690/996f3b7c-310d-4da2-9b9e-a4487c3b448f) | ![Capture d’écran 2023-10-05 à 16 25 19](https://github.com/odoo/odoo/assets/80679690/a8bda994-cef8-4516-b2e0-a6152bf9bbb1)  |

Reply:
Moving this part out for the eCommerce team https://github.com/odoo/odoo/pull/137953